clang-tools-extra/clangd/refactor/Rename.cpp

```diff
@@ -605,12 +607,12 @@ findAllSelectorPieces(llvm::ArrayRef<syntax::Token> Tokens,
       auto PieceCount = SelectorPieces.size();
       if (PieceCount < NumArgs &&
           isMatchingSelectorName(Tok, Tokens[Index + 1], SM,
-                                 Sel.getNameForSlot(PieceCount))) {
+                                 Name.getNamePieces()[PieceCount])) {
```

Comment:
nit: can store the pieces as a variable around L600 to make this look a bit cleaner.